frontend/src/Pages/Students.tsx

```diff
@@ -0,0 +1,340 @@
+        mutate();
+    };
+
+    const hanldleEditUser = () => {
+        editUserModal.current?.close();
+        resetModal();
+        mutate();
+    };
+
+    const handleDeleteUserCancel = () => {
+        deleteUserModal.current?.close();
+        resetModal();
+    };
+
+    const handleResetPasswordCancel = (msg: string, err: boolean) => {
+        const state = err ? ToastState.error : ToastState.success;
+        if (msg === '' && !err) {
+            resetUserPasswordModal.current?.close();
+            resetModal();
+            return;
+        }
+        showToast(msg, state);
+        resetModal();
+    };
+
+    const handleDisplayTempPassword = (psw: string) => {
+        setTempPassword(psw);
+        resetUserPasswordModal.current?.close();
+        showUserPassword.current?.showModal();
+        showToast('Password Successfully Reset', ToastState.success);
+    };
+
+    const handleShowPasswordClose = () => {
+        showUserPassword.current?.close();
+        setTempPassword('');
+        resetModal();
+    };
+
+    const handleChange = (newSearch: string) => {
+        setSearchTerm(newSearch);
+        setPageQuery(1);
+    };
+
+    return (
+        <AuthenticatedLayout title="Students" path={['Users']}>
+            <div className="flex flex-col space-y-6 overflow-x-auto rounded-lg p-4">
+                <div className="flex justify-between">
+                    <div className="flex flex-row gap-x-2">
+                        <SearchBar
+                            searchTerm={searchTerm}
+                            changeCallback={handleChange}
+                        />
+                        <DropdownControl
+                            label="order by"
+                            callback={setSortQuery}
+                            enumType={{
+                                'Name (A-Z)': 'name_last asc',
+                                'Name (Z-A)': 'name_last desc',
+                                'Account Created ↓ ': 'created_at desc',
+                                'Account Created ↑ ': 'created_at asc'
+                            }}
+                        />
+                    </div>
+
+                    <div className="tooltip tooltip-left" data-tip="Add User">
+                        <button
+                            className="btn btn-primary btn-sm"
+                            onClick={() => addUserModal.current?.showModal()}
+                        >
+                            <UserPlusIcon className="h-4" />
+                        </button>
+                    </div>
+                </div>
+                <table className="table">
+                    <thead>
+                        <tr className="border-gray-600">
+                            <th className="flex">
+                                <span>Name</span>
+                            </th>
+                            <th>Username</th>
+                            {/*<th>Role</th>*/}
```

Comment:
Rather than comment this out, lets just delete this line of code. Not relevant anymore